src/scripts/components/globe-navigation/globe-navigation.styl

```diff
@@ -7,13 +7,18 @@
   z-index: 1
   display: flex
 
-.compassIcon
-  padding-right: emCalc(16px)
-  padding-left: emCalc(25px)
+.compass
+  margin-right: emCalc(16px)
+  margin-left: emCalc(25px)
+  width: 24px
+  height: 24px
 
-.compassIcon svg, .downloadIcon svg
-  fill: $textWhite
+.compass svg
+  fill: #202B33
```

Comment:
What color is thta? Wouldn't it be better if the icon defines the default fill color itself and we overwrite only on hover?